pkgs/by-name/im/immersed/package.nix

```diff
@@ -7,16 +7,20 @@
 }:
 let
   pname = "immersed";
-  version = "10.5.0";
+  version = "10.6.0";
 
   sources = rec {
     x86_64-linux = {
-      url = "https://web.archive.org/web/20240909144905if_/https://static.immersed.com/dl/Immersed-x86_64.AppImage";
-      hash = "sha256-/fc/URYJZftZPyVicmZjyvcGPLaHrnlsrERlQFN5E98=";
+      url = "https://web.archive.org/web/20241229041449/https://static.immersed.com/dl/Immersed-x86_64.AppImage";
+      hash = "sha256-PUV3Nt2HpFZkJ7dof2gjC1Ayey0HYRoG2dZqTZQgDpo=";
```

Comment:
I got this hash error. Perhaps you have a mistake to generate packages' hash?

Command:
```bash
NIXPKGS_ALLOW_UNFREE=1 nix run github:Pandapip1/nixpkgs/e7574b7f1d5ce8d983941cbfb1bf72074bf48ad4#immersed --impure
```

Error:
```txt
error: hash mismatch in fixed-output derivation '/nix/store/s8nfwrwaq4wz5dz6w6j48jika0k5b6h9-Immersed-x86_64.AppImage.drv':
         specified: sha256-PUV3Nt2HpFZkJ7dof2gjC1Ayey0HYRoG2dZqTZQgDpo=
            got:    sha256-u07QpGXEXbp7ApZgerq36x+4Wxsz08YAruIVnZeS0ck=
error: 1 dependencies of derivation '/nix/store/cvjqjqv1fjic364q1j3g7bnwjga82sgn-immersed-10.6.0-extracted.drv' failed to build
error: 1 dependencies of derivation '/nix/store/r1q336yz7h62hd5a1iwhsbjl0lx8zczf-immersed-10.6.0-init.drv' failed to build
```

`nix-info`:
```txt
$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.6.67, NixOS, 24.11 (Vicuna), 24.11.711934.4005c3ff7505`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.24.10`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```